### PR TITLE
EAP 7.2 model changes

### PR DESCRIPTION
**Pull request by zebrik**

### CODE REVIEW

**Comment by zebrik** [comment]
Hi @spriadka , can you please review this?

File: basic/src/test/java/org/jboss/hal/testsuite/test/configuration/container/IIOPTestCase.java [hunk 1]
```diff
@@ -177,10 +179,13 @@ public void expectErrorForInvalidHighWaterMarkValue() throws Exception {
 
     @Test
     public void highWaterMarkValueTest() throws Exception {
+        String highWaterMark = "high-water-mark";
```
Comment (spriadka) [inline basic/src/test/java/org/jboss/hal/testsuite/test/configuration/container/IIOPTestCase.java]: I prefer to have the ``highWaterMark`` and ``securityDomain`` as a constant to keep consistent with the rest of the test suite

**Review by spriadka — COMMENTED** [review]
Just a minor nit-picking, otherwise LGTM, please squash and merge

**Comment by zebrik** [comment]
Thank you for the review I have added requested changes.

**Pull request merged**